### DISCUSSION TITLE
GitHub Action: Pull request 'development' object/data, aka linked issues

### DISCUSSION THREAD

**Discussion by Ismoh** [community/community#34172] [Pull Requests] [answered]
Hey folks, I am searching since two days for any opportunity to get the "development" section data in a pull request. (Top right corner, where you can set linked issues)
Whether by gh cli, GitHub API nor by action workflow or any other way to get the "development" information, nothing of those provides a get endpoint to request those information.

Is there anyone out there, who can answer my question?

Background info:
If there is a pr on my repo, I want to set a linked issue in "development" and depending on this issue, labels are cloned to the pr. Pr workflow now checks the labels and increases the SemVer by specific predefined labels.
But for god sakes reason, there is no way to request this information. Or better to say, I found nothing on any github docs that exist.

Would be awesome, if anyone can give me a hint!
Thanks a ton!
Have a good one and happy coding!

**Comment by fidou** [comment]
same here, I am also unable to find the api anywhere. A nice feature for automation

**Reply by Ismoh** [reply]
If you find anything regarding this, let me know!

**Comment by Ismoh** [comment]
Looks like I may found something:

https://stackoverflow.com/a/60066508/3493998

https://github.blog/changelog/2020-02-04-linked-search-qualifier-for-finding-issues-connected-to-pull-requests/

https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+author%3A%40me+archived%3Afalse+linked%3Apr

Now, it's up to us to create API call which fits.

If I find time, I am going to release a marketplace action for this!

**Reply by Ismoh** [reply]
@fidou

**Reply by Ismoh** [reply]
A simple example:
https://api.github.com/search/issues?q=linked%3Apr%20author%3AIsmoh

Marketplace action will be released soon!

**Comment by Ismoh** [comment, marked as answer]
https://github.com/Ismoh-Games/find-linked-issues
